sklearn/model_selection/tests/test_search.py

```diff
@@ -1403,9 +1405,7 @@ def test_search_cv_results_none_param():
             est_parameters,
             cv=cv,
         ).fit(X, y)
-        assert_array_equal(
-            grid_search.cv_results_["param_random_state"], [0, float("nan")]
-        )
+        assert_array_equal(grid_search.cv_results_["param_random_state"], [0, None])
```

Comment:
`random_state` is documented to accept an integer or `None`, but not `float` - so I think the new output looks more correct?

Reply:
Yup. `random_state` should not be a float.